src/azul/service/avro_pfb.py

```diff
@@ -167,8 +167,13 @@ def _add_missing_fields(cls, name: str, object_: MutableJSON, schema):
         None is the default value, but because of https://github.com/DataBiosphere/azul/issues/2370
         this isn't currently reflected in the schema.
         """
-        object_schema = one(f for f in schema['fields'] if f['name'] == 'object')
-        entity_schema = one(e for e in object_schema['type'] if e['name'] == name)
+        if schema['type'] == 'record':
```

Comment:
Don't understand why this is in the commit `Include all file_base fields in related_files`. PL item maybe?

Reply:
Avro PFB change factored out into `Fix: Avro PFB doesn't handle missing fields in dict nested in inner entities`  commit